packages/vm/src/cache.rs

```diff
@@ -253,6 +253,14 @@ where
 {
 }
 
+unsafe impl<A, S, Q> Send for Cache<A, S, Q>
+where
+    A: BackendApi + 'static,
+    S: Storage + 'static,
+    Q: Querier + 'static,
+{
+}
+
```

Comment:
I thought about this, but didn't implement it because it's not really needed. It doesn't harm to have it, though.

Reply:
Oh, I see now that it's needed for thread spawning. Nice.